src/sql/expertise_outstanding_by_id.sql

```diff
@@ -0,0 +1,15 @@
+SELECT
```

Comment:
There are a bunch of different ways we could approach this. Let's connect in the AM to talk it out.